src/editor/PxtoneClient.cpp

```diff
@@ -366,7 +366,7 @@ void PxtoneClient::processRemoteAction(const ServerAction &a) {
                     [this, uid](const Overdrive::Remove &s) {
                       m_controller->applyRemoveOverdrive(s, uid);
                     },
-                    [this, uid](const Delay::Set &s) {
+                    [this, uid](const ZDelay::Set &s) {
```

Comment:
Tiny thing, could you rename this to like, "DelayEffect" instead maybe?

Reply:
> @yuxshao In this latest commit, I had a conflict between some Cocoa stuff and the `Delay` namespace in RemoteAction. For now I have renamed it to "ZDelay" -- that's a stupid name and I want to come up with a better name or workaround. Unfortunately, it's not a preprocessor definition so I can't just `#undef` it like I've had to do 3 or 4 times for Windows stuff thus far. Lmk

i brought it up here
sure

Reply:
should i rename overdrive too?

Reply:
resolved on da cord